app/views/home/landing.html.erb

```diff
@@ -7,7 +7,8 @@
       {
         contactEmail: "accounts@idseq.net",
         showBulletin: <%= @show_bulletin %>,
-        browserInfo: <%= @browser_info.to_json.html_safe %>
+        browserInfo: <%= @browser_info.to_json.html_safe %>,
+        signInStrategy: location.href.match(/auth0/) ? 'AUTH0' : 'DEVISE',
```

Comment:
nit: could you move this line into the `HomeController#landing` method like `@sign_in_strategy`? Just want to keep these erb files very slim on logic